modules/appeals_api/spec/factories/higher_level_reviews.rb

```diff
@@ -40,75 +26,38 @@
     id { SecureRandom.uuid }
     api_version { 'V2' }
     auth_headers do
-      JSON.parse(File.read(::Rails.root.join(
-        *'/modules/appeals_api/spec/fixtures/decision_reviews/v2/valid_200996_headers_extra.json'.split('/')
-      ).to_s)).transform_values(&:strip)
-    end
-    form_data do
-      JSON.parse(File.read(::Rails.root.join(
-        *'/modules/appeals_api/spec/fixtures/decision_reviews/v2/valid_200996_extra.json'.split('/')
-      ).to_s))
+      FixtureHelpers.fixture_as_json('decision_reviews/v2/valid_200996_headers_extra.json')
+                    .transform_values(&:strip)
```

Comment:
I am not sure why we use `transform_values` on `extra` headers for Supplemental Claims and Higher-Level Reviews and nowhere else, but if I remove it, it introduces spacing changes in the PDF generator specs, so I've left it in for now

Reply:
I ended up moving these changes into #13001 because they put this above the changed lines limit